### PR TITLE
Fix typo in XRInterface comment

### PR DESCRIPTION
**Pull request by camren-m**
Corrects a typo (the -> they) in xr_interface.h on the XRInterface class' comment.

### CODE REVIEW

**Comment by AThousandShips** [comment]
Thank you for your contribution

This is a very small change for a whole PR, I think this should instead be part of a larger style and spelling fix PR that are some times done, along with applying spell checking directly

**Comment by cmumme** [comment]
> Thank you for your contribution
> 
> This is a very small change for a whole PR, I think this should instead be part of a larger style and spelling fix PR that are some times done, along with applying spell checking directly

Hi, thank you for your quick response. Similar PRs about the same size have been merged in the past. (e.g. #83792) Despite this, merging this into a larger PR as has also been done in cases like #85141 would also make sense.

File: servers/xr/xr_interface.h [hunk 1]
```diff
@@ -43,7 +43,7 @@ struct BlitToScreen;
 	The idea is that we subclass this class, implement the logic, and then instantiate a singleton of each interface
 	when Godot starts. These instances do not initialize themselves but register themselves with the AR/VR server.
 
-	If the user wants to enable AR/VR the choose the interface they want to use and initialize it.
+	If the user wants to enable AR/VR they choose the interface they want to use and initialize it.
```
Comment (fire) [inline servers/xr/xr_interface.h:46]:
```suggestion
	If the user wants to enable AR/VR, they can choose the interface they want to use and initialize it.
```

**Review by dsnopek — APPROVED** [review]
Looks good to me!

**Pull request merged**